src/components/Home-Repos.js

```diff
@@ -60,6 +61,17 @@ class HomeRepos extends Component {
           </div>
         </IconContext.Provider>
         <a className="Repos-repo-name" href={ node.url } target="_blank" rel="noopener noreferrer">{node.name}</a>
+
+        <IconContext.Provider value={ {} }>
+          <div className="Repos-repo-icon">
+            <FaStar />
+          </div>
+        </IconContext.Provider>
+
+        <div className="Repos-repo-stars">
+          {node.stargazers.totalCount}
```

Comment:
Let's move this to a function because its more then one level deep. We want to protect against `cannot get some-value of undefined errors`. Follow the same patter I used in `Home-Stats.js`. e.g.

```
starsCount = () => {
    if (this.props.userData && this.props.userData.starredRepositories) {
      const { totalCount } = this.props.userData.starredRepositories;
      return totalCount !== null ? totalCount : '?';
    }
    return '?';
  }
```